File: 31/Auto.py
from Veicoli import Veicoli 
class Auto(Veicoli):
    def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero,capienzaBagagliaio,cilindrata,categoria) -> None:
        super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
        self.capienzaBagagliaio = capienzaBagagliaio
        self.cilindrata = cilindrata
        self.categoria = categoria
       
    def __str__(self) -> str:
        return self.tipo+"\t"+self.targa+"\t"+str(self.codice)+"\t"+self.modello+"\t"+self.marca+"\t"+str(self.costoGiornaliero)+"\t"+str(self.capienzaBagagliaio)+"\t"+str(self.cilindrata)+"\t"+self.categoria+"\t-"+"\t-"+"\t-"

File: sim07/src/Auto.java
public class Auto extends Veicolo{
    int cilindrata;
    double Capienza_bagagliaio;
    public Auto(int codice, String tipo, String targa, String modello, String marca, int cilindrata,
            double capienza_bagagliaio) {
        super(codice, tipo, targa, modello, marca);
        this.cilindrata = cilindrata;
        Capienza_bagagliaio = capienza_bagagliaio;
    }
    @Override
    public String toString() {
        return tipo+targa+codice+modello+marca+costo_giornaliero+cilindrata+Capienza_bagagliaio+" - "+" - "+" - ";
    }
}
